Tests/FeatherWidgets.TestIntegration/Navigation/LanguageSelectorWidgetTests.cs

```diff
@@ -141,6 +146,173 @@ public void LanguageSelectorWidget_CurrentLanguageNotIncluded()
             this.AssertLanguageLinks(pageContent, expectedLinkes, notExpectedLinkes);
         }
 
+        [Test]
+        [Multilingual]
+        [Category(TestCategories.Navigation)]
+        [Author(FeatherTeams.Team7)]
+        [Description("Verifies language selector is included in detail view of content items")]
```

Comment:
To test all possible cases for detail view you can create a news 1 with en and tr translation and news 2 with en translation only. Then load the detail view for news 1 and verify that the tr link will load the detail translation of news 1. Then load the detail view for news 2 and verify that the tr link will load the list view. This is the expected behavior because when the detail view does not have translation the list view in tr will be loaded. (just in case check the behavior in Release_8_0_Fixes ) The same applies for LanguageSelectorWidget_CurrentLanguageNotIncludedInDetailsViewOfContentItems test.